src/components/JobBoards/JobBoardItem/JobBoardItem.js

```diff
@@ -313,7 +313,7 @@ export class JobBoardItem extends React.Component {
   renderBoardTags () {
     const { jobBoard } = this.props
     const tags = [
-      ...jobBoard.tagsCategory.map(c => c.name),
+      jobBoard.tagsCategory,
```

Comment:
should use jobBoard.tagsCategory.name
or it displays `[Object object]`
![screen shot 2018-05-23 at 11 43 46](https://user-images.githubusercontent.com/9588637/40413482-93184552-5e7e-11e8-8826-2d0815b0165a.png)
![screen shot 2018-05-23 at 11 43 27](https://user-images.githubusercontent.com/9588637/40413484-93428c22-5e7e-11e8-84f2-92ecdbf20243.png)

Reply:
Thanks